rust/src/ssh/ssh.rs

```diff
@@ -203,13 +224,24 @@ impl SSHState {
                         parser::MessageCode::NewKeys => {
                             hdr.flags = SSHConnectionState::SshStateFinished;
                             if ohdr.flags >= SSHConnectionState::SshStateFinished {
-                                unsafe {
```

Comment:
Why are we changing the default ?

SSH by default did like `SSH_HANDLE_ENCRYPTION_BYPASS` here : APP_LAYER_PARSER_NO_INSPECTION | APP_LAYER_PARSER_NO_REASSEMBLY | APP_LAYER_PARSER_BYPASS_READY

Reply:
it only did it when stream.bypass was enabled, in the default settings (with a fresh suricata.yaml), stream.bypass is disabled and as a result SSH connections should not be bypassed.